### DISCUSSION TITLE
Patroni service failing

### DISCUSSION THREAD

**Discussion by ganamur1983** [patroni/patroni#3176] [Q&A]
Hi All,

OS: RHEL 9
Postgres 15

I am getting attached error while starting patroni services on DB node 1 and DB node 2. Please review the attached yml file and error log and suggest.

I tried to remove blank spaces but still same error.
[Patroni_error.txt](https://github.com/user-attachments/files/17266706/Patroni_error.txt)
[Patroni_config_file.txt](https://github.com/user-attachments/files/17266707/Patroni_config_file.txt)

**Comment by CyberDem0n** [comment]
The error pretty clearly says that the config file isn't correctly formatted.
Please use YAML linter (search for it in the internet, there are plenty available, some of them even online).
You can also take [sample config files](https://github.com/patroni/patroni/blob/master/postgres0.yml) from the repo as an example of properly formatted YAML files.
